types/global.d.ts

```diff
@@ -17,11 +17,11 @@ import {
   EthereumSignMessage,
   EthereumSignTypedDataTypes,
 } from '@trezor/connect-web';
+import type { Provider } from '@metamask/network-controller';
 import {
   OffscreenCommunicationTarget,
   TrezorAction,
-} from 'shared/constants/offscreen-communication';
-import type { Provider } from '@metamask/network-controller';
+} from '../shared/constants/offscreen-communication';
```

Comment:
Previously, madge would find this file, and try to import this file, which doesn't exist at the given location, and would just ignore it (it marks it as `skipped`, which this PR no longer allows)

Side note: Our typescript lint process seems to ignore this file.